_includes/manuals/3.1/1.2_release_notes.md

```diff
@@ -24,7 +24,291 @@ Existing installations should start thinking about a migration plan.
 
 Note that this is only support to run Foreman and Foreman Proxy themselves on EL7. Managing EL7 systems remains supported. See [the RFC](https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008) for more information.
 
-### Release Notes
+### Release notes for 3.1.0
+#### Foreman
+* foreman-develop-package-release fails after removing react-intl \- [#33800](https://projects.theforeman.org/issues/33800)
+* Handle different Windows installation types \- [#33769](https://projects.theforeman.org/issues/33769)
+* Make dynflow-sidekiq@\* services part of foreman.service \- [#33723](https://projects.theforeman.org/issues/33723)
+* Allow foreman plugins to use webpack \- [#33626](https://projects.theforeman.org/issues/33626)
+* Rename UNDI iPXE filename to end with zero \- [#33584](https://projects.theforeman.org/issues/33584)
+* Use gray background only for overview tab on Host detail page \- [#33423](https://projects.theforeman.org/issues/33423)
+* Fix Nightly React test failures \- [#33265](https://projects.theforeman.org/issues/33265)
+* After creating new host group or Http proxy, page doesn't redirect to main landing page. \- [#33177](https://projects.theforeman.org/issues/33177)
+* servcie network restart on EL8 doesn't work \- [#33092](https://projects.theforeman.org/issues/33092)
+* Add scoped search definition for arf reports that finds reports for hosts owned by current user \- [#33025](https://projects.theforeman.org/issues/33025)
```

Comment:
This isn't what the fix ended up doing, message should be modified (it stuck out since arf reports isn't part of foreman core)